src/Twig/Extension/DebugExtension.php

```diff
@@ -41,4 +41,12 @@ public function dump(\Twig_Environment $env, $value)
 
         return stream_get_contents($dump);
     }
+
+    /**
+     * @return string
+     */
+    public function getName() : string
+    {
+        return "guzzle";
```

Comment:
I tested. Really with old twig this problem exists. But my suggestion is to write here `return get_class($this);`, because `guzzle` is too generic.

Reply:
Yeah, this is better.